src/integrationTest/java/io/aiven/kafka/connect/jdbc/oracle/AbstractOracleIT.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2024 Aiven Oy and jdbc-connector-for-apache-kafka project contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.jdbc.oracle;
+
+import javax.sql.DataSource;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.HashMap;
+import java.util.Map;
+
+import io.aiven.kafka.connect.jdbc.AbstractIT;
+
+import oracle.jdbc.pool.OracleDataSource;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.oracle.OracleContainer;
+import org.testcontainers.utility.DockerImageName;
+
+public class AbstractOracleIT extends AbstractIT {
+
+    public static final String DEFAULT_ORACLE_TAG = "slim-faststart";
+    private static final DockerImageName DEFAULT_ORACLE_IMAGE_NAME =
+            DockerImageName.parse("gvenzl/oracle-free")
+                    .withTag(DEFAULT_ORACLE_TAG);
+    @Container
+    public static final OracleContainer ORACLE_CONTAINER = new OracleContainer(DEFAULT_ORACLE_IMAGE_NAME);
```

Comment:
This should match the convention set in `AbstractPostgresIT`:

1. Should not be `static` (otherwise, gets reused across tests)
2. Visibility should be `protected` instead of `public`
3. Name shouldn't be all-caps (due to being non-static)

```suggestion
    protected final OracleContainer oracleContainer = new OracleContainer(DEFAULT_ORACLE_IMAGE_NAME);
```